newt/vendor/mynewt.apache.org/newt/util/util.go

```diff
@@ -42,6 +42,7 @@ import (
 )
 
 var Verbosity int
+var PrintBuildCmds bool
```

Comment:
Hmm... the godep tool seems to be pretty much broken, at least for our use case.  There are two options that will work:

1. Perform a complete revendoring of newt.
2. Just copy the util.go changes by hand.

I think option 2 is preferred here, since we don't want to upgrade all the dependencies when the current ones are working.  The problem is that the Godeps.json file will no longer be correct, but since the tool doesn't work for us anyway, I don't see much value in this file!  So, I would just copy your updated util.go file to util/util.go and commit it.

For the record, here is how you would do a full revendoring:
1. `rm -rf Godeps vendor`
2. `git commit -a -m 'revendor newt'`
3. `godep save`
4. `git add Godeps vendor`
5. `git commit -a -m 'revendor newt'`
6. Use `git rebase -i` to fixup the two revendor commits

I think we need to look into a tool other than godep to do our vendoring.  It looks like godep hasn't been updated for a while.